pom.xml

```diff
@@ -391,7 +424,7 @@
         <dependency>
             <groupId>org.testfx</groupId>
             <artifactId>openjfx-monocle</artifactId>
-            <version>8u76-b04</version>
+            <version>jdk-12.0.1+2</version>
```

Comment:
jdk-11+26 version did not work. Had some missing methods. Upgrading resolved issues.